src/constants/apiPaths.ts

```diff
@@ -3,7 +3,7 @@ const API_PATHS = {
   product: 'https://.execute-api.eu-west-1.amazonaws.com/dev',
   order: 'https://.execute-api.eu-west-1.amazonaws.com/dev',
   import: 'https://.execute-api.eu-west-1.amazonaws.com/dev',
-  bff: 'https://.execute-api.eu-west-1.amazonaws.com/dev'
+  bff: 'https://299bl3bv5g.execute-api.eu-west-1.amazonaws.com/dev'
```

Comment:
I guess we should have used API_PATHS.product

Reply:
well, actually in initial code was bff commented, so who knows :)